staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1407,6 +1407,7 @@ type DeveloperConfiguration struct {
 	MemoryOvercommit       int               `json:"memoryOvercommit,string,omitempty"`
 	NodeSelectors          map[string]string `json:"nodeSelectors,omitempty"`
 	UseEmulation           bool              `json:"useEmulation,string,omitempty"`
+	CPUAllocationRatio     float64           `json:"cpuAllocationRatio,string,omitempty"`
```

Comment:
For consistency, would it make sense to align this CPU ratio with how the `MemoryOvercommit` option works?

MemoryOvercommit appears to scale the requested guest memory based on an integer where 100 is is 1:1. So if i ask for 1024Mi of memory in the guest with a MemoryOvercommit of 120, i'll get 853Mi actually requested. And likewise if I change the MemoryOvercommit to 80 then i'll get 1280Mi.


We could represent this similar for CPU with a `CPUOvercommit` option that behaves the same way. Functionally this is pretty much what you have now, it's just presented slightly different.

Reply:
Let me how it works, but basically, yes, it doesn't have to be a float

Reply:
I think an integer that represents personage won't work here. We will need to get to a large number that won't make sense. Normally, the overcommit ratio is around 1 pcpu to 16 virtual. 
I think `ratio` would be understood better. I can make it an integer, if need, but the user would a set an int in the config map anyway..,

Reply:
OSP also represents this as "cpu_allocation_ratio":  cpu_allocation_ratio = guest_vCPUs / cores

Reply:
I like your ratio logic more than what we have for memory overcommit and agree that it's easier to understand. Functionally, i think the math checks out either way though if we went with something similar to memory commit for cpus.

Examples

```
requested cores = cpu_allocation_ratio * guest_vCPUs
```

is equivalent to 

```
requested cores = (100 / cpu_overcommit) * guest_vCPUs
```

Regardless, i'm for keeping it the  way you have it with the ratio now. There's nothing intuitive around the memory overcommit int.